### ISSUE TITLE
No documentation is existant and unable to run the code with dotnet core 2.2

### ISSUE THREAD

**Issue by nberic** [daxnet/irony#13]
Where could I find Irony .NET framework (or preferably dotnet core) documentation? Codeplex site states that `This project migrated to https://github.com/IronyProject/`, however I could not find any documentation on both github and codeplex sites (it seems outdated and non-buildable for current versions of .NET framework). On the other hand, the `A modified version of the Irony project (https://irony.codeplex.com) with .NET Core support ` doesn't provide any documentation at all. There is just an example that doesn't seem to run. Also, some examples of languages seem to be incorrect as they fail build. 

Is the framework still maintained or it got obsolete and it is revival state. Seems quite difficult to learn anything about it from the examples (although the examples are quite similar to the BNF). 

Are there any substitutions?

** Got redirected from stackoverflow.com **

**Comment by daxnet** [comment]
@nberic 
The project is still alive and it should be https://github.com/IronyProject/Irony.  In the beginning, I migrated the original implementation from Codeplex simply because codeplex was going to be closed and also because I was on a project which utilized Irony. After that, one of my follower (if I could remember) started the above Irony repo and took over the maintenance of the project, from my perspective, in particular for this repo here, I think I should archive it because I don't have too much spare time in maintaining this project, so, if you are interested in Irony, the great compiler generator in .NET, please go to that repo. I am sorry for the confusion.
By the way, I got myself started with Irony simply by reading an article from codeproject.com. I believe you can do the same thing, just search Irony on codeproject.com and see if there is any articles that can help.
Thanks.

**Comment by nberic** [comment]
@daxnet 
Sorry for the long-waited reply. The project is great itself, and I have read the articles on codeproject.com. However, the provided code mostly doesn't work anymore. 
The site from the URL also lacks any real documentation. Also, it gets really difficult to implement a feature of an app with Irony when any deeper concepts and usages are nowhere to be found  and sometimes it seams that some features of Irony are missing which other parsers have. However, it is still great. Thank you for the migration. Too bad this great idea didn't live enough.

**Comment by daxnet** [comment]
@nberic 
Yes, you are absolutely right. As Irony is not maintained by its original author anymore, most of the usages of this library is somehow experimental and once there comes an issue, it is hard to solve. Anyways, this this a great library that I use in some of my personal projects, like https://github.com/daxnet/logo-sharp. If you are interested, you could go through the repo and see if it helps.
Thanks!

**Comment by nberic** [comment]
@daxnet 
Great. Thank you.
